lib/symlink/index.js

```diff
@@ -51,22 +51,24 @@ function symlink(outFolder, opt) {
       srcPath = path.relative(file.base, srcPath);
     }
 
-    // TODO: make DRY with .dest()
-    var dirMode = number(opt.dirMode, file);
-    var writeFolder = path.dirname(file.path);
-
-    fo.mkdirp(writeFolder, dirMode, onMkdirp);
+    // Because fs.symlink does not allow atomic overwrite option with flags, we
+    // delete and recreate if the link already exists and overwrite is true.
+    if (file.flag === 'w') {
+      // TODO What happens when we call unlink with windows junctions?
```

Comment:
@afinnell Do you happen to know, are have an opportunity to check, what happens when we call `fs.unlink` on a JUNCTION link on Windows? Hoping it just deletes the link, not the target...